client/packages/common/src/intl/locales/en/common.json

```diff
@@ -66,6 +67,7 @@
   "heading.settings-display": "Display settings",
   "heading.settings-sync": "Synchronisation",
   "heading.shipments-to-be-picked": "Shipments to be picked",
+  "heading.no-patient-record": "Patient not available",
```

Comment:
This one is also fairly blunt! maybe an 'oops' or 'oh no!' or 'Error' ?

Reply:
Changed it to:

Header: Oops! Something's gone wrong. (re-using existing key)
Text: Can't access detailed information for this patient.